content/tickets.rst

```diff
@@ -136,15 +138,42 @@ Below is a list of used tags:
   - These are tasks that don’t require knowledge of the product or operating system 'know-how' in advance.
 
 
+Ticket components
```

Comment:
I can make a note that this area is being revisited.
And list them out.

Could be a reminder to go through this... maybe going through ticket approach could be a June meeting agenda ?